.github/workflows/dev_module_build.yml

```
@@ -533,6 +533,15 @@ jobs:
         run: |
           task run -v
 
+      - uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: resources_from_failed_tests
+          retention-days: 21
+          path: /tmp/e2e_failed__*
+          if-no-files-found: ignore
+retention-days: 2
```

The default retention period is 15 days. I think that for this action we only need 2
```suggestion
        with:
          name: resources_from_failed_tests
          retention-days: 21
          path: /tmp/e2e_failed__*
          if-no-files-found: ignore
          retention-days: 2
```